src/css/popup.css

```diff
@@ -246,6 +247,30 @@ textarea.materialize-textarea:focus:not([readonly]) {
 	top: 18px;
 }
 
+.collection-item {
+	padding-top: 5px !important;
+	padding-bottom: 5px !important;
+	padding-right: 5px !important;
+	width: 100% !important;
+}
+
+.remove-margin {
+	padding-top: 0px !important;
+	padding-bottom: 0px !important;
+	margin-bottom: 0px !important;
+}
+
+#custom-domain-label {
+	font-size: 1.3rem;
+	width: 290px;
```

Comment:
If the label width is not hard coded, the window expands horizontally to accommodate long text, even with the overflow: hidden and text-overflow: ellipsis attributes. 290 is a value which keeps the width consistent with the other windows in JTI.